### PR TITLE
[HTSN-557] create subscribe modal

### PR DESCRIPTION
**Pull request by hngerebara**
Subscribe modal based on mock design https://app.zeplin.io/project/5d581fcb1db0449b12b76763/screen/5d65212b684fa927fe85cae9

Screenshot 2019-10-03 at 10 40 33 AM

### CODE REVIEW

**Comment by josh-padnick** [comment]
Hi Hope, without reviewing the code here's some feedback from your screenshot:

1. I realize this image is zoomed, but even at normal zoom, Grunty still looks blurry. Can you create a much higher resolution version of him?
image

2. Fix spacing for better "proximity". To understand more about the concept of proximity see https://vwo.com/blog/crap-design-principles/ and scroll down to the section on proximity. Basically, either create more space between each grouping of text, or less spacing between the headline and the underlying text. The first option is probably the right one.

image

**Comment by hngerebara** [comment]
Thanks, Josh for the feedback
The image is to be replaced. I requested Sam to export the actual image which she just did. That's why I indicated WIP

**Comment by hngerebara** [comment]
Screenshot 2019-10-04 at 2 23 07 AM

**Comment by josh-padnick** [comment]
Thanks, Hope. The image looks better now. Is that what you were intending to show with the screenshot? It looks like the sections' spacing is still off. Also, the middle section repeats the phrase "Learn More" twice.

File: _includes/subscribe-modal.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<div class="modal fade" id="subscribe-success" tabindex="-1" role="dialog" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered subscribe-modal " role="document">
+        <div class="modal-content text-center">
+            <div class="modal-header">
+                <button type="button" class="close" onclick="window.history.pushState({path: window.location.pathname },'',window.location.pathname)"  data-dismiss="modal" aria-label="Close">
```
Comment (brikis98) [inline _includes/subscribe-modal.html]: I'm having a brain fart... Can you remind me why we do a `pushState` on the modal?

File: _includes/subscribe-modal.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                <img src="/assets/img/guides/img-subscribe.png" alt="subscribe_image" /><br />
+            </div>
+            
+            <div class="modal-body">
+                <h2 class="no-anchor">Subscribe to Gruntwork</h2>
+                <ul>
+                    <li>
+                        <h3 class="modal-title no-anchor">Gruntwork Service Catalog</h3>
+                        <p>Get access to over 300,000 lines of reusable, battle-tested, production-grade infrastructure code for AWS and GCP.
+                            <a href="#">Coming Soon</a>
```
Comment (brikis98) [inline _includes/subscribe-modal.html]: Coming soon?

Comment (brikis98) [reply _includes/subscribe-modal.html]: Shouldn't this be a "Learn More" link that goes to the IaC Lib page?

File: _includes/subscribe-modal.html (hunk 1 above)
Comment (hngerebara) [reply _includes/subscribe-modal.html]: To remove the parameter from the route which in this case is `subscribe_modal = 1` and avoid reloading that page

File: _includes/subscribe-modal.html (hunk 2 above)
Comment (hngerebara) [reply _includes/subscribe-modal.html]: Ooh I thought this was going to be linked with the service catalog currently being worked on. So Sam suggested we change that to coming soon. I'll fix that now

File: _includes/subscribe-modal.html (hunk 1 above)
Comment (brikis98) [reply _includes/subscribe-modal.html]: Ah, right. Sorry for forgetting. Perhaps add a comment with that explanation?

**Pull request merged**